### PR TITLE
Issue 320: add scripts for publishing new website to staging and apache

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

- script for staging changes to a staging-site repo
- steps for how to staging changes for reviews
- script for pushing changes to apache

### CODE REVIEW

File: site/scripts/publish-website.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
```
Comment (lucperkins) [inline site/scripts/publish-website.sh]: For some of these variables that are set in both `publish-website.sh` and `staging-website.sh`, I'd recommend putting those in `common.sh` and adding `source scripts/common.sh` at the top. Keeps the scripts more DRY.

Comment (lucperkins) [inline site/scripts/publish-website.sh]: Not a blocker, though. LGTM otherwise.

Comment (sijie) [inline site/scripts/publish-website.sh]: good suggestion. I will make the changes.

Comment (sijie) [inline site/scripts/publish-website.sh]: @lucperkins I moved the common variables to the common.sh

**Pull request closed without merging**